tests/test_ipr/test_upload.py

```diff
@@ -166,6 +166,8 @@ def compare_sections(section1, section2):
                         item[subitem].pop(key, None)
                     if isinstance(item[subitem], list):
                         [subsubitem.pop(key, None) for subsubitem in item[subitem]]
+                        if item[subitem] != []:
+                            item[subitem] = sorted(item[subitem], key=lambda d: str(d))
```

Comment:
I'm not sure the if statement is actually doing anything, but I do not think it matters.